CONTRIBUTING.md

```diff
@@ -16,128 +16,158 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
+- this will format Python code and notebooks
 
-If you are adding new examples, see below.
+### Enforced naming
 
-### Organization
+Directories and files must follow:
 
-PRs to add new examples should consider which type of example to add:
+- naming must be logical
+- directories under `tutorials` must be words separated by hyphens
+- tutorial workflows (and workflow files) use the naming convention `run-tutorial-*initials*`, where *initials* is the initials of the words
 
-* `examples` is for general examples using AML and should run `code` examples
-* `notebooks` is for general example notebooks using AML and should be interactive
-* `tutorials` is for end to end tutorials using AML
+### Unenforced naming
 
-### Enforced naming
+- `environment_name` = "framework-example|tutorial" e.g. "pytorch-example"
+- `experiment_name` = "logical-words-example|tutorial" e.g. "hello-world-tutorial"
+- `compute_name` = "compute-defined-in-setup-workspace.py" e.g. "gpu-K80-2"
+- `ws = Workspace.from_config()`
+- `dstore = ws.get_default_datastore()`
+- `ds = Dataset.File.from_files(...)`
+- `env = Environment.from_*(...)`
+- `src = ScriptRunConfig(...)`
+- `run = Experiment(ws, experiment_name).submit(src)`
 
-PRs must follow the following naming conventions:
+### Adding a new ________?
 
-* naming must be logical
-* under `notebooks` use the naming convention *scenario-framework-etc-compute*, where *scenario* is one of ["train", "deploy", "score", "dprep"]
-* directories under `tutorials` must be words separated by hyphens
-* tutorial workflows (and workflow files) use the naming convention `run-tutorial-*initials*`, where *initials* is the initials of the words
+Thinking of contributing a new example? Read this first!
 
-### Testing
+#### Tutorials
 
-PRs must include necessary changes to any testing to ensure:
+A tutorial is an end-to-end example accomplishing something significant or teaching how to scale up and out in the cloud. A tutorial **must** have an excellent `README.md` file in its directory, following conventional markdown syntax, explaining:
 
-* `run-examples` runs on every push and PR to `main` (with changes to examples) and runs all examples under `examples/`
-* `run-notebooks` runs on every push and PR to `main` (with changes to notebooks) and runs all examples under `notebooks/`
-* `run-tutorial-initials` must be tested at least daily and on PR to `main` (with changes to the tutorial)
-* `cleanup` runs daily and cleans up AML resources for the testing workspace
-* `smoke` runs hourly and on every push and PR to `main` and performs sanity checks
+- required prerequisites
+- any one-time setup needed by the user
+- any other setup instructions
+- overview of files in the tutorial
+- relevant links
 
-### Miscellaneous
+Tutorials are often, but not required to be, a series of ordered Jupyter notebooks. All Jupyter notebooks must utilize notebook features (i.e. be interactive, have explanation in markdown cells, etc).
```

Comment:
Should probably say something about how you expect meaningful explanations in markdown cells.